livemaps-api/APIFunctions/RoomData.cs

```diff
@@ -71,7 +67,7 @@ public static class RoomData
                     if (useAtlas)
                     {
                         features = await MapsService.FetchFeaturesFromAtlas(buildingConfig.DatasetId, buildingConfig.SubscriptionKey);
-                        await featureMapref.UploadTextAsync(JsonConvert.SerializeObject(features));
+                        await featureMapref.UploadAsync(BinaryData.FromString(JsonConvert.SerializeObject(features)), overwrite: true);
```

Comment:
Can you let BinaryData do the conversion?

```suggestion
                        await featureMapref.UploadAsync(BinaryData.FromObjectAsJson(features), overwrite: true);
```

Reply:
Ditto for everything else here